reactive/vpe_router.py

```diff
@@ -35,8 +35,10 @@ def validate_config():
     except Exception as e:
         remove_state('vpe.configured')
         set_state('blocked', 'validation failed: %s' % e)
-    else:
```

Comment:
It would probably be less confusing to just move the `else:` code inside the `try` block at the bottom.